bin/all_sky_search/pycbc_plot_kde_vals

```diff
@@ -34,20 +36,30 @@ if args.signal_file:
     signal_kde = signal_data['data_kde'][:]
 template_data = h5py.File(args.template_file, 'r')
 template_kde = template_data['data_kde'][:]
-param0 = template_data[args.param[0]][:]
-if len(args.param) > 1:
-    param1 = template_data[args.param[1]][:]
+param_arrays = [template_data[param][:] for param in args.param]
+
+kde_arrays = {
+    'signal_kde': signal_kde,
+    'template_kde': template_kde,
+    'ratio_kde': signal_kde / template_kde,
+}
+kde_values = kde_arrays[args.which_kde]
```

Comment:
As you don't use the variable `kde_arrays` anywhere else, can you just do this instead? 
```
kde_values = {
    'signal_kde': signal_kde,
    'template_kde': template_kde,
    'ratio_kde': signal_kde / template_kde,
}[args.which_kde]
```